### PR TITLE
fix: session-based cookie auth is not supported on swagger docs

### PR DESCRIPTION
**Pull request by tan-zx**
## Problem

Closes https://github.com/opengovsg/postmangovsg/issues/1769

Currently, the "try it out" is misleading. 

When we do authentication via cookies, we do so by checking if the request is coming from an authenticated session , rather than looking at the cookie attached in the request. 

As such, have removed it.

Furthermore, have improved the documentation to make clear the format of the bearer token.

### CODE REVIEW

**Comment by stanleynguyen** [comment]
This definition [is correct](https://swagger.io/docs/specification/authentication/cookie-authentication/) though, please dont remove it. Maybe add a description for this security schema to clarify instead

**Review by stanleynguyen — CHANGES_REQUESTED** [review]
This definition [is correct](https://swagger.io/docs/specification/authentication/cookie-authentication/) though, please dont remove it. Maybe add a description for this security schema to clarify instead

File: backend/src/core/routes/index.ts [hunk 1]
```diff
@@ -170,7 +170,9 @@ export const InitV1Route = (app: Application): Router => {
    *     bearerAuth:
    *       type: http
    *       scheme: bearer
+   *       bearerFormat: username_versionNumber_apiKey
    *     cookieAuth:
+   *       description: while this description (is correct)[https://swagger.io/docs/specification/authentication/cookie-authentication/], do note that cookie authentication is currently not supported for "try it out" requests due to browser security restrictions.
```
Comment (stanleynguyen) [inline backend/src/core/routes/index.ts:175]: It is supported though, we're just using the wrong cookie name. Should be `postmangovsg-v2`
Screenshot 2022-10-02 at 12 17 55 PM

Comment (stanleynguyen) [reply backend/src/core/routes/index.ts:175]: It was working when I "tried it out"

Comment (tan-zx) [reply backend/src/core/routes/index.ts:175]: Ohh I copied that line from the page you linked and it didn't work when I tried. 

When I checked the cookie name from my browser, it didn't have the v2, but let me try again when I have the time!

Comment (tan-zx) [reply backend/src/core/routes/index.ts:175]: where did you get the `COOKIE_NAME` in the screenshot from? mine is here:

image

and I tried it here (which didn't work):

image

Comment (tan-zx) [reply backend/src/core/routes/index.ts:175]: oh I found it on production, that one has the `v2`

Comment (tan-zx) [reply backend/src/core/routes/index.ts:175]: hmm but trying out production using that cookie still gives me a Internal Server Error 
image

Comment (tan-zx) [reply backend/src/core/routes/index.ts:175]: from the website [you linked](https://swagger.io/docs/specification/authentication/cookie-authentication/) 
image

Comment (stanleynguyen) [reply backend/src/core/routes/index.ts]: I guess we can remove it then. We're narrowing this swagger doc to just API later on anw so

Comment (tan-zx) [reply backend/src/core/routes/index.ts]: by remove, you mean remove the `cookieAuth` right? will do

Comment (tan-zx) [reply backend/src/core/routes/index.ts]: done

**Review by stanleynguyen — APPROVED** [review]
